qp_fastp_minimap2/qp_fastp_minimap2.py

```diff
@@ -0,0 +1,139 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2020, Qiita development team.
+#
+# Distributed under the terms of the BSD 3-clause License License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+import pandas as pd
+
+from os import environ
+from os.path import basename, join
+from glob import glob
+from itertools import zip_longest
+
+from qiita_client import ArtifactInfo
+from qiita_client.util import system_call
+
+
+QC_REFERENCE_DB = environ["QC_REFERENCE_DB"]
+
+FASTP_CMD = ('fastp -l 100 -i %s -I %s -w {nprocs} '
+             '-o {out_dir}/%s -O {out_dir}/%s')
+COMBINED_CMD = ('fastp -l 100 -i %s -I %s -w {nprocs} --stdout | '
+                'minimap2 -ax sr -t {nprocs} {database} - -a | '
+                'samtools fastq -@ {nprocs} -f 12 -F 256 '
+                '-1 {out_dir}/%s -2 {out_dir}/%s')
+FASTP_CMD_SINGLE = ('fastp -l 100 -i %s -w {nprocs} '
+                    '-o {out_dir}/%s')
+COMBINED_CMD_SINGLE = ('fastp -l 100 -i %s -w {nprocs} --stdout | '
+                       'minimap2 -ax sr -t {nprocs} {database} - -a | '
+                       'samtools fastq -@ {nprocs} -f 12 -F 256 '
+                       '-1 {out_dir}/%s')
+
+
+def get_dbs_list():
+    folder = QC_REFERENCE_DB
+
+    # skip human database
+    return [basename(f) for f in glob(f'{folder}/*.mmi') if 'human' not in f]
```

Comment:
Makes sense, although the new sequence preprocessing workflow would need this. Any thoughts on how we could facilitate access there?

Reply:
Got it! IMOO out of the scope of this PR but it will depend on how the only admin commands are added; perhaps having a single list/table of plugins, versions, parameters that are available for admins would be the way to go.

Reply:
Makes a lot of sense, thanks for the explanation!